projects/portfolio/portfolio-webpage-2.md

```diff
@@ -24,16 +24,16 @@ A great portfolio page will be a valuable way to showcase your skills as you loo
 
 ## Project Requirements
 
-- [ ] As you review the topic outline, commit your code in Git and push to GitHub after each bolded section. You should write clear + informative commit messages in command form ("Add line about technical skills") for each commit.
+- [ ] As you review the topic outline, commit your code in Git and push to GitHub after each bolded section. You should write clear + informative commit messages in command form (For example: "Add line about technical skills") for each commit.
 
 ---
 
 ### Part 1 - Incorporate SEO into your portfolio webpage
 
 **Open latest version of Portfolio Project**
 
-- [ ] Using the command line, practice pulling the latest version from your remote git repo.
-- [ ] Open your portfolio project in your text editor application using your command line.
+- [ ] Using the command line, practice pulling the latest version from your remote git repo. (If you have only worked on it locally, it will show no changes and that you are up to date)
```

Comment:
This is a great opportunity for a screenshot of what that might look like (the terminal output).